src/renderer/services/auto-updater.service.ts

```diff
@@ -35,11 +55,68 @@ export class AutoUpdaterService {
         });
 
         this.progressService.show(this.downloadProgress$, true);
-        
         return promise;
     }
 
     public quitAndInstall() {
         this.ipcService.sendLazy("install-update");
     }
+
+    public getLastAppVersion(): string {
+        return this.configurationService.get("last-app-version");
+    }
+
+    public setLastAppVersion(){
+        lastValueFrom(this.getAppVersion()).then(v => this.configurationService.set("last-app-version", v));
+    }
+
+    private async getChangelog(): Promise<Changelog> {
+        try {
+        if (this.cacheChangelog) {
```

Comment:
You don't need a try catch here beacause its a Promise
Just do as we did in the `getChangelogVersion` function